### ISSUE TITLE
`@clerk/upgrade` does not seem to work via yarn

### ISSUE THREAD

**Issue by richardasymmetric** [clerk/javascript#3316]
### Preliminary Checks

- [X] I have reviewed the documentation: [https://clerk.com/docs](https://clerk.com/docs)
- [X] I have searched for existing issues: [https://github.com/clerk/javascript/issues](https://github.com/clerk/javascript/issues)
- [X] I have not already reached out to Clerk support via email or Discord (if you have, no need to open an issue here)
- [X] This issue is not a question, general help request, or anything other than a bug report directly related to Clerk. Please ask questions in our Discord community: [https://clerk.com/discord](https://clerk.com/discord).

### Reproduction

[https://example.com/not_applicable](https://example.com/not_applicable)

### Publishable key

pk_not_applicable

### Description

**Steps to reproduce:**

1. yarn dlx @clerk/upgrade

**Expected behavior:**

It should do whatever it's supposed to do

**Actual behavior:**
It errors

```
$ yarn dlx @clerk/upgrade
➤ YN0000: · Yarn 4.1.1
➤ YN0000: ┌ Resolution step
➤ YN0058: │ marked-terminal@https://github.com/jescalan/marked-terminal.git#commit=44f5ab42076e16937f56d645d3d7264675558ea1: Packing the package failed (exit code 1, logs can be found here: /tmp/xfs-303b25cc/pack.log)
➤ YN0000: └ Completed in 8s 790ms
➤ YN0000: · Failed with errors in 8s 810ms
$ cat /tmp/xfs-303b25cc/pack.log
Packing marked-terminal@https://github.com/jescalan/marked-terminal.git#commit=44f5ab42076e16937f56d645d3d7264675558ea1 from sources
Using npm for bootstrap. Reason: found npm's "package-lock.json" lockfile
```

### Environment

```Shell
System:
    OS: Linux 6.8 Ubuntu 24.04 LTS 24.04 LTS (Noble Numbat)
    CPU: (16) x64 AMD Ryzen 7 2700 Eight-Core Processor
    Memory: 18.82 GB / 31.29 GB
    Container: Yes
    Shell: 5.2.21 - /bin/bash
  npmPackages:
    @clerk/nextjs: ^5.0.5 => 5.0.5
    @headlessui/react: 1.7.19 => 1.7.19
    @heroicons/react: ^1.0.6 => 1.0.6
    @hookform/resolvers: ^3.3.4 => 3.3.4
    @mailchimp/mailchimp_transactional: 1.0.59 => 1.0.59
    @next/eslint-plugin-next: ^14.2.3 => 14.2.3
    @ngrok/ngrok: ^1.2.0 => 1.2.0
    @sentry/nextjs: ^7.110.1 => 7.110.1
    @tailwindcss/forms: ^0.5.7 => 0.5.7
    @tailwindcss/typography: ^0.5.12 => 0.5.12
    @testing-library/dom: ^9.3.4 => 9.3.4
    @testing-library/jest-dom: ^6.4.2 => 6.4.2
    @testing-library/react: ^14.3.1 => 14.3.1
    @testing-library/user-event: ^14.5.2 => 14.5.2
    @types/jest: ^29.5.12 => 29.5.12
    @types/luxon: ^3.4.2 => 3.4.2
    @types/mailchimp__mailchimp_transactional: ^1.0.9 => 1.0.9
    @types/node: ^18.19.31 => 18.19.31
    @types/node-fetch: ^2.6.11 => 2.6.11
    @types/pg: ^8.11.5 => 8.11.5
    @types/react: ^18.2.78 => 18.2.78
    autoprefixer: ^10.4.19 => 10.4.19
    axios: 0.21.4 => 0.21.4
    babel-jest: ^29.7.0 => 29.7.0
    concurrently: ^8.2.2 => 8.2.2
    dotenv: ^16.4.5 => 16.4.5
    eslint: ^8.57.0 => 8.57.0
    eslint-config: ^0.3.0 => 0.3.0
    eslint-config-next: ^14.2.3 => 14.2.3
    eslint-config-prettier: ^9.1.0 => 9.1.0
    eslint-plugin-jest-dom: ^5.2.0 => 5.2.0
    eslint-plugin-simple-import-sort: ^12.0.0 => 12.0.0
    eslint-plugin-testing-library: ^6.2.2 => 6.2.2
    husky: ^9.0.11 => 9.0.11
    jest: ^29.7.0 => 29.7.0
    jest-environment-jsdom: ^29.7.0 => 29.7.0
    jest-watch-select-projects: ^2.0.0 => 2.0.0
    jest-watch-typeahead: ^2.2.2 => 2.2.2
    jose: ^5.2.4 => 5.2.4
    jwt-decode: ^4.0.0 => 4.0.0
    knex: ^3.1.0 => 3.1.0
    lint-staged: ^15.2.2 => 15.2.2
    luxon: ^3.4.4 => 3.4.4
    msw: 1.0.1 => 1.0.1
    next: ^14.2.3 => 14.2.3
    node-fetch: ^2.6 => 2.7.0
    node-mocks-http: ^1.14.1 => 1.14.1
    npm-run-all: ^4.1.5 => 4.1.5
    pg: ^8.11.5 => 8.11.5
    plaid: ^23.0.0 => 23.0.0
    postcss: ^8.4.38 => 8.4.38
    postcss-import: ^16.1.0 => 16.1.0
    prettier: ^3.2.5 => 3.2.5
    prettier-plugin-tailwind-css: ^1.5.0 => 1.5.0
    react: ^18.3.1 => 18.3.1
    react-dom: ^18.3.1 => 18.3.1
    react-hook-form: ^7.51.3 => 7.51.3
    react-plaid-link: ^3.5.1 => 3.5.1
    react-tooltip: ^5.26.3 => 5.26.3
    swr: 2.2.5 => 2.2.5
    tailwindcss: ^3.4.3 => 3.4.3
    ts-brand: ^0.1.0 => 0.1.0
    ts-node: ^10.9.2 => 10.9.2
    tsconfig-paths: ^4.2.0 => 4.2.0
    typescript: ^5.4.5 => 5.4.5
    typescript-eslint: ^7.7.0 => 7.7.0
    yup: ^1.4.0 => 1.4.0
```

**Comment by LekoArts** [comment]
Hi, thanks for the issue!

I tried `yarn dlx @clerk/upgrade` on an example repo of mine and it works:

```shell
clerk-nextjs-app-quickstart on  main [⇣!?] via ⬢ v20.11.1 took 2.7s
➜ yarn dlx @clerk/upgrade
➤ YN0000: · Yarn 4.0.1
➤ YN0000: ┌ Resolution step
➤ YN0085: │ + @clerk/upgrade@npm:1.0.6, @alcalzone/ansi-tokenize@npm:0.1.3, @babel/code-frame@npm:7.24.2, and 230 more.
➤ YN0000: └ Completed in 5s 441ms
➤ YN0000: ┌ Fetch step
➤ YN0013: │ 233 packages were added to the project (+ 18.66 MiB).
➤ YN0000: └ Completed in 0s 345ms
➤ YN0000: ┌ Link step
➤ YN0000: └ Completed in 0s 553ms
➤ YN0000: · Done in 6s 367ms

 █▀▀ █   █▀▀ █▀█ █▄▀   █ █ █▀█ █▀▀ █▀█ ▄▀█ █▀▄ █▀▀
 █▄▄ █▄▄ ██▄ █▀▄ █ █   █▄█ █▀▀ █▄█ █▀▄ █▀█ █▄▀ ██▄

✓ File scan complete. See results below!

Navigation Instructions:
Navigate through items with ↑ and ↓ arrow keys. Expand the details of any item with space bar. ↓ key on the last item goes to the next page, ↑
on the first item goes to the previous page. To exit this interface, use "control + c".

[...]
```

Some questions for you:

1. Are you able to provide a repository where this always happens for you?
2. If not, can you provide the error logs you're getting?
3. Is this also happening with `npx @clerk/upgrade`?

If you have yarn installed you also have npm installed, so you can use `npx @clerk/upgrade` to unblock yourself (if it works). You don't necessarily have to use yarn 👍

**Comment by richardasymmetric** [comment]
I can't seem to make a reproduction in codespace, I ended up doing it manually anyhow, and I'll close this for now.

**Comment by clerk-cookie** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
